include/openssl/sslerr.h

```diff
@@ -257,6 +259,7 @@ int ERR_load_SSL_strings(void);
 # define SSL_F_TLS_CONSTRUCT_CTOS_HELLO                   356
 # define SSL_F_TLS_CONSTRUCT_CTOS_KEY_EXCHANGE            357
 # define SSL_F_TLS_CONSTRUCT_CTOS_KEY_SHARE               470
+# define SSL_F_TLS_CONSTRUCT_CTOS_MAXFRAGMENTLEN          547
```

Comment:
# define SSL_F_SSL_GENERATE_SESSION_ID                    547

Reply:
the number 547 is used twice ?

Reply:
Good catch ! Fixed.
@richsalz :  I remember duplicated Ids were breaking the build, but not today ... looks like a internal toolchain regression .